icl-cluster-profiles/profiles/ci.yaml

```diff
@@ -0,0 +1,53 @@
+# Generated from Kapitan target ci
+
+calico_endpoint_to_host_action: ACCEPT
+calico_ipip_mode: Never
+calico_network_backend: bird
+calico_vxlan_mode: Never
+containerd_registries_mirrors:
+- mirrors:
+  - capabilities:
+    - pull
+    - resolve
+    host: https://dockerhubcache.caas.intel.com
```

Comment:
Let's move specific configuration to the runner.

Reply:
Fixed

Reply:
Sorry, another attempt